buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -100,7 +100,21 @@ public void run() {
          *
          * See https://github.com/netty/netty/issues/3888.
          */
-        final int defaultMinNumArena = NettyRuntime.availableProcessors() * 2;
+        int defaultMinNumArena = NettyRuntime.availableProcessors() * 2;
+
+        // Respect MALLOC_ARENA_MAX if set.
+        final String mallocArenaMaxStr = System.getenv("MALLOC_ARENA_MAX");
```

Comment:
Also note, if the system property is set we can avoid this code path. Maybe execute conditionally?

Reply:
which system property you talk about ?

Reply:
Nitsan probably means if we have any way to opt out if the env variable is set - to have a custom behaviour